caver/patcher.py

```diff
@@ -21,9 +21,9 @@ def get_path() -> Path:
         file_dir = Path(__file__).parent.parent
     return file_dir.joinpath("caver")
 
-def patch_files(patch_data: dict, output_dir: Path, progress_update: Callable[[str, float], None]):
+def patch_files(patch_data: dict, output_dir: Path, platform: str, progress_update: Callable[[str, float], None]):
```

Comment:
you know what, I think the `CSPlatform` enum should be defined in this repo instead, and imported in RDV. much cleaner than using magic strings